google-cloud-firestore/src/test/java/com/google/cloud/firestore/it/ITQueryAggregationsTest.java

```diff
@@ -0,0 +1,856 @@
+package com.google.cloud.firestore.it;
+
+import static com.google.cloud.firestore.AggregateField.average;
+import static com.google.cloud.firestore.AggregateField.sum;
+import static com.google.cloud.firestore.LocalFirestoreHelper.autoId;
+import static com.google.cloud.firestore.LocalFirestoreHelper.map;
+import static com.google.cloud.firestore.it.TestHelper.await;
+import static com.google.cloud.firestore.it.TestHelper.isRunningAgainstFirestoreEmulator;
+import static com.google.common.truth.Truth.assertThat;
+import static java.util.Arrays.asList;
+import static org.junit.Assert.assertThrows;
+import static org.junit.Assume.assumeFalse;
+import static org.junit.Assume.assumeTrue;
+
+import com.google.cloud.firestore.*;
+import com.google.common.base.Preconditions;
+import java.util.Map;
+import java.util.concurrent.ExecutionException;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public class ITQueryAggregationsTest {
+  private Firestore firestore;
+
+  @Before
+  public void setUpFirestore() {
+    // TODO: stop using emulator for these tests once prod is ready.
+    firestore = FirestoreOptions.newBuilder().setHost("localhost:8080").build().getService();
```

Comment:
I think this line needs to be updated for production launch.

Reply:
Good catch. Fixed.